https://issues.redhat.com/browse/ACM-26586 (#8446)
open-cluster-management/rhacm-docs: release_notes/known_issues_continuity.adoc

Backup and restore known issues and limitations are listed here, along with workarounds if they are available.

[#baremetalhost-not-backed-up]
== The _BareMetalHost_ resources are not backed up by default

> You can automatically add hosts to the  host inventory by using `BareMetalHosts`. By default, your `BareMetalHosts` are not included in your backups. 

Reviewer: ```suggestion
You can automatically add hosts to the host inventory by using `BareMetalHosts`. By default, your `BareMetalHosts` are not included in your backups. 
```

> If you implement a central infrastructure management practice, the `BareMetalHosts` add hosts to a host inventory. By default, your `BareMetalHosts` are not included in your backups. 

Reviewer 2: This sentence is a bit odd as written. Maybe something like this?

```suggestion
You can add hosts to host inventory automatically using `BareMetalHosts` 
```

I think dropping the CIM reference is fine. The main doc for adding hosts with BMH is [here](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.15/html/clusters/cluster_mce_overview#auto-add-host-host-inventory) if it's helpful
